core/sqf/monitor/linux/zclient.cxx

```diff
@@ -53,6 +53,9 @@
 // The monitors register their znodes under the cluster znode
 #define ZCLIENT_CLUSTER_ZNODE               "/cluster"
 
+// zookeeper connection retry count
+#define ZOOKEEPER_RETRY_COUNT                3
```

Comment:
Wouldn't it be better to make this an environment variable or monitor config setting that can be changed instead of a hardcoded retry count of 3.